src/components/Header.tsx

```diff
@@ -23,7 +23,10 @@ import { Link, useHistory } from "react-router-dom";
 const useStyles = makeStyles((theme: Theme) =>
   createStyles({
     appBar: {
-      paddingLeft: theme.spacing(7) + 1
+      paddingLeft: theme.spacing(7) + 1,
+      position: "sticky",
```

Comment:
@nk183 how does the UI appear on minimizing the page with these changes?

Reply:
hi @HerbertYiga  it working the same 

https://user-images.githubusercontent.com/58779460/115845636-ac440a80-a43e-11eb-8dd4-eef3a0dc51ed.mp4

Reply:
cc @ibacher